support-frontend/assets/pages/supporter-plus-landing/supporterPlusRouter.tsx

```diff
@@ -18,7 +18,7 @@ import { SupporterPlusInitialLandingPage } from './twoStepPages/firstStepLanding
 import { SupporterPlusCheckout } from './twoStepPages/secondStepCheckout';
 import { ThreeTierLanding } from './twoStepPages/threeTierLanding';
 
-validateWindowGuardian(window.guardian);
+parseAppConfig(window.guardian);
```

Comment:
Presume this is necessary to match checkout for consistency if you go here direct?

Reply:
As `supporterPlusRouter` still uses `window.guardian` all this will do is error if we put the `window.guardian` object in an invalid state.